static/bidder-info/adtonos.yaml

```diff
@@ -0,0 +1,24 @@
+endpoint: https://exchange.adtonos.com/bid/{{.PublisherID}}
+maintainer:
+  email: support@adtonos.com
```

Comment:
Sent email for verification.

Reply:
Thank you, I believe a response has been sent back accordingly.

Reply:
email verified. thanks!